scripts/gen_vimdoc.py

```diff
@@ -239,6 +239,7 @@
 annotation_map = {
     'FUNC_API_FAST': '{fast}',
     'FUNC_API_CHECK_TEXTLOCK': 'not allowed when |textlock| is active',
+    'FUNC_API_LUA_ONLY' : 'only allowed from lua'
```

Comment:
`Lua` is a proper noun and should be capitalized (I included that in my last suggestion).

```suggestion
    'FUNC_API_LUA_ONLY' : 'only allowed from Lua'
```

Reply:
@gpanders  can you help me to run this .py file